_internal/utils/use-swr-config.ts

```diff
@@ -2,10 +2,10 @@ import { useContext } from 'react'
 import { defaultConfig } from './config'
 import { SWRConfigContext } from './config-context'
 import { mergeObjects } from './helper'
-import { FullConfiguration, Cache } from '../types'
+import { FullConfiguration, Cache, CacheValue } from '../types'
 
 export const useSWRConfig = <
-  T extends Cache = Cache
+  T extends Cache = Map<string, CacheValue>
```

Comment:
I believe we no longer want to use `Map`: https://github.com/vercel/swr/pull/1961/files#r872298911

Reply:
Yes, since the default cache is actually a `Map`.   
https://github.com/vercel/swr/blob/9a0328fffdeb8ca0eaf42bc88f318f7e48a3b245/_internal/utils/config.ts#L44